securedrop-client/debian/changelog-buster

```diff
@@ -1,3 +1,9 @@
+securedrop-client (0.2.2+buster) unstable; urgency=medium
```

Comment:
Version bump in https://github.com/freedomofpress/securedrop-client/pull/1160 is marked as temporary, we should either bump the client version and add the changelog as part of 1160 or in a separate PR, prior to merging this changelog addition here.

Reply:
Good point. I'll _remove_ the version bump in https://github.com/freedomofpress/securedrop-client/pull/1160, as well as remove the version bump in this PR. For nightlies, that'll be good enough: the consolidation logic additions we wish to QA will be available in apt-test main.